src/NUnitFramework/framework/Internal/Extensions/TypeExtensions.cs

```diff
@@ -8,5 +8,20 @@ internal static class TypeExtensions
     {
         public static bool ImplementsIComparable(this Type type) =>
             type?.GetInterface("System.IComparable") != null;
+
+        public static bool IsSortable(this Type type)
+        {
+            if (!type.ImplementsIComparable())
+                return false;
+
+            if (TypeHelper.IsTuple(type) || TypeHelper.IsValueTuple(type))
+            {
+                foreach (var typeArg in type.GetGenericArguments())
+                    if (!typeArg.IsSortable())
+                        return false;
```

Comment:
```suggestion
                foreach (var typeArg in type.GetGenericArguments())
                {
                    if (!typeArg.IsSortable())
                        return false;
                }
```
Personally, I would like to see curly braces here as it is more than one line, but I don't know what/if there are any agreed style guidelines for nunit. I don't see a [csharp_prefer_braces](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0011#csharp_prefer_braces) entry in .editorconfig which means it can be anything. For nunit.analyzers we use the `when_multiline` option.

Reply:
I would create a separate issue as adding that will very likely require changes to other places to comply.
Yes, a consistent coding standard improves readability.

Reply:
Sounds good to me, thanks!